scripts/flights2.py

```diff
@@ -679,35 +671,67 @@ async def _request_async(session: niquests.AsyncSession, name: str, /) -> io.Byt
         raise NotImplementedError(msg)
 
 
-def _write_rezip(input_dir: Path, buf: io.BytesIO, /) -> Path:
+def _write_zip_to_parquet(input_dir: Path, buf: io.BytesIO, /) -> Path:
```

Comment:
Side-by-side size comparison:

![image](https://github.com/user-attachments/assets/6513a598-9f5a-434e-a50b-14a963558be4)

Reply:
### Related
These document the progression through `.zip` -> `.csv.gz` -> `.parquet`:
- [feat(perf): Async requests, use `gzip` instead of `.zip`](https://github.com/vega/vega-datasets/pull/645/commits/aede7f6aa63f0a1c559db5618fc26536e9931d33)
- [refactor: Reorganize, add `_write_rezip_async` doc](https://github.com/vega/vega-datasets/pull/645/commits/402c2b0f42e1241dd9e9a3ccf4f40c53cb6b6261)
- [feat(perf): Store `.parquet` instead of `.csv.gz`](https://github.com/vega/vega-datasets/pull/645/commits/c668fd7b166f6ff7bffa769d96245076002aa125)